readme.md

```diff
@@ -53,6 +53,14 @@ The following dependencies need to be installed on your system:
 			* MSVC v142 - VS 2019 C++ ARM64 build tools
 			* C++ ATL for v142 build tools (ARM64)
 
+When installing Visual Studio, you can import the config:
```

Comment:
I think there is a large benefit to instructing everyone to do this the same way. Can you move these instructions up to before the list of components for Visual studio please?